net-proxy/trojan/trojan-1.11.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{3_4,3_5,3_6,3_7} )
+
+inherit cmake-utils python-any-r1 systemd
+
+DESCRIPTION="An unidentifiable mechanism that helps you bypass GFW"
+HOMEPAGE="https://github.com/trojan-gfw/${PN}"
+SRC_URI="https://github.com/trojan-gfw/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="mysql test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-libs/boost-1.54.0:=
+	>=dev-libs/openssl-1.0.2:=
+	mysql? ( dev-db/mysql-connector-c:= )
+"
+DEPEND="
+	${RDEPEND}
+	test? ( net-misc/curl ${PYTHON_DEPS} )
```

Comment:
Without testing, why's curl needed in src_test? Does it require active internet connection? Does your sandbox allow the tests to finish? :)

Reply:
The test will listen on `localhost`. `curl` only connects to the `localhost`.